development/playbooks/setup-repositories/setup-repositories.yaml

```diff
@@ -9,10 +9,23 @@
         name: theforeman.operations.puppet_repositories
       vars:
         foreman_puppet_repositories_version: "8"
+      when:
+        - ansible_distribution_major_version == '9'
```

Comment:
Should this be when the certificate source is the installer instead?

Reply:
Yepp, but this detail is not passed to the "setup repos" playbook, and I was lazy